src/grammar/statement/select.rs

```diff
@@ -80,7 +80,11 @@ impl Statement for Select {
             buffer.push("*");
         }
         buffer.push("FROM");
-        buffer.push(format!("`{}`", some!(self.table)));
```

Comment:
In your current implementation, the resulting query might end up without any tables. Previously, `some!` for checking for this.

Reply:
It would be nice to have a test for this.